build/istio/values.yaml

```diff
@@ -5,7 +5,7 @@
 #! These values cannot be changed later when rendering cf-for-k8s templates.
 #! Values related to CF should NOT be in this file.
 
-istio_version: 1.7.3
+istio_version: 1.8.0
```

Comment:
There shouldn't be any reason to move to the latest patch version. 1.7.x is compatible with 1.8.x. It's not like 1.7.3 was the latest patch release of 1.7.x anyways.

Reply:
Yeah, we'll be going full steam ahead to `1.8.4`! 😌